### ISSUE TITLE
Shared Folders not working on kernel 4.0.5-1

### ISSUE THREAD

**Issue by putztzu** [rasa/vmware-tools-patches#63]
kernel 4.0.5-1 is the default kernel for openSUSE Tumbleweed.
This should be a tested and supported kernel.

Compiling Tools 9.9.3.
No error thrown compiling vmhgfs.

Shared Folders "No Device" found when attempting to mount.
All other Tools capabilities seem to be working properly.

**Comment by rasa** [comment]
Fixed by #64? If not, please reopen.

**Comment by putztzu** [comment]
Just checked on an openSUSE Tumbleweed, shared folders is broken after doing a git pull to ensure running latest main branch. Earlier today, I updated an Ubuntu running kernel 3.13.x and an openSUSE 13.2 running kernel 3.16.x with no problems.

So, it appears there is probably a current problem running kernel 4.x.
From my TW

uname -a
Linux linux-cm8y 4.1.4-1-desktop #1 SMP PREEMPT Mon Aug 3 18:25:04 UTC 2015 (a37e14f) x86_64 x86_64 x86_64 GNU/Linux

**Comment by putztzu** [comment]
For anyone who wants to set up a test environment, I'm using openSUSE Tumbleweed.
I also expect that Fedora Rawhide likely runs with a bleeding edge kernel release.
Both the above distribute and run bleeding edge kernels by default so you don't have to do anything unusual to get a bleeding edge kernel installed, is default.

For anyone who wants to set up an openSUSE Tumbleweed, these are the main requirements...
From the TW portal page
https://en.opensuse.org/openSUSE:Tumbleweed_installation#Repositories
You can download an install ISO, the following link is to the x64 iso
http://download.opensuse.org/tumbleweed/iso/openSUSE-Tumbleweed-DVD-x86_64-Current.iso

When installing openSUSE TW, I recommend choosing "advanced" so that you do not accept all the defaults (although you can do so). You then have the ability to choose non-default options like the DE.

 I recommend installing a lightweight DE like LXDE or XFCE which will run easily with 2GB RAM instead of the default KDE which I've found requires twice as much RAM.

Once the install has been completed, you must install a special package that contains legacy net tools to run VMware-tools-patches(VMware should fix their Tools to use ip instead). Run the following command from a root console (su). I included "git" just so it isn't forgotten

zypper install net-tools-deprecated git

Your system is now prepped to clone this project's repo and install the VMware tools patches.
You can verify tools has been successfully installed by re-sizing your Guest Window,  tools will enable the Guest to auto-resize the display to fill the window. But, when you test for the shared folders mount you will find it's not created.

**Comment by putztzu** [comment]
Update: I found an error message I don't remember seeing before... "This program could not find a valid path to  the gcc binary."

Have verified gcc is properly installed, am speculating for now that gcc 5.1.1 is not supported by the VMware tools compiler. Am looking for a way to downgrade the compiler.

As I described earlier, this is a new development since Tools was partially working. 
At the moment I will need to do some investigation to verify my suspicion that the problem is the gcc version.
